src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+
+The map vision should not use “#00FF00 on #000000” color scheme for purpose of reducing eye fatigue. Consider dark dark gray background as a default for areas that are not space exposed. Preferably implement a way to set custom colors for your UI.
+
+### Cameras
+
+Cameras cover almost all of the crew areas, places where most people work and exist within, to monitor their wellbeing and the state of the station. They should not be nearly free to build, and should require an autolathe producible board, some steel, LV wire.
+
+They should not be more durable than a person - a hard enough knock should deactivate it for a small period of time, while sustaining enough damage will keep it offline until repaired. A non-violent option for keeping cameras offline should be available, using cloth and/or gauze makes it impossible to see through a camera, taking a relatively long time to cover it completely (3 or 5 second doafter, balanced by it being a “stealth” option). A camera disabled in such a way should still be shown as active for AI and warden. Though it should have no motion detection cone and convey such explicitly.
+
+Power is required for cameras to work at full capacity, reducing their vision to half or third of their normal range when it is not available. Though power being out is a bigger issue for AI itself than just the cameras.
+
+## Borgs - Sync and control
+
+The AI has no way to affect anything physical (at least, a shackled one - not malf). Its primary way to do anything in that regard is borgs, and directing the crew. While the crew has own concerns, like air, health and such, borgs are hardier and are almost always available for AI to use.
+
+Borgs can opt into synchronizing their laws with AI, gaining map or camera view from AI (todo: detail the incentives). This does not make them less susceptible to ion storm law corruption, but the AI can fix the laws by rewriting them to its own at some cost.
+
+Emagging should forcefully but silently unlink a borg from the AI if it’s synced. If it’s not, AI is not relevant.
+
+Crew can cut AI control wire (multiple required?) in borg frames to prevent them from being able to sync to the AI, and letting the crew modify their laws once again (or just resetting to what they had before AI overriding them - including previously present ion laws, if there’s no way to set laws of a borg).
+
+## Assorted ideas
+
+- Paperwork management UI that AI gets *instead* of being able to interact with fax machines. It would list all papers received and let the player read them. It would also allow editing (if not stamped). Allows sending the same way a fax can.
```

Comment:
I like the AI able to do paperwork. I could see the RP of the station AI being assigned as a beurocrat in charge of keeping documents in check. and mabey MALF AI could get a "cybersun pen" upgrade that allows them to edit stamped documents like syndicates. I could imagine how chaotic that would be. even more so if the AI could eg: intercept all Faxes to CentComm and start acting as CentComm and giving superior orders. but that would only reallly play out on MRP/HRP servers. cause who uses faxes in LRP.